core-java-modules/core-java-jar/src/test/java/com/baeldung/jar/JarAppUnitTest.java

```diff
@@ -3,10 +3,14 @@
 import org.junit.Assert;
 import org.junit.jupiter.api.Test;
 
+import java.util.regex.Pattern;
+
 public class JarAppUnitTest {
 
     @Test
     public void findClassTest(){
-        Assert.assertTrue(JarApp.findObjectMapperClass().endsWith("jackson-databind-2.13.3.jar"));
+        Pattern databindPattern = Pattern.compile(".*jackson-databind-(\\d)+\\.(\\d)+\\.(\\d)\\.jar$");
+
+        Assert.assertTrue(databindPattern.matcher(JarApp.findObjectMapperClass()).matches());
```

Comment:
This change is in no way related to this PR, but it's broken and causing the checks to fail.

The actual break is because Jackson is now 2.15.2 instead of 2.13.3, but I've done it this way so that it's resilliant to all future version changes.